app/models/derived_variables/sales_log_variables.rb

```diff
@@ -17,7 +17,20 @@ def set_derived_fields!
       self.homonth = hodate.month
       self.hoyear = hodate.year
     end
-    self.deposit = value if outright_sale? && mortgage_not_used?
+
+    if outright_sale? && mortgage_not_used?
+      self.deposit = value
+    elsif outright_sale? && mortgage_use_unknown?
```

Comment:
This might not be quite right either, per https://dluhcdigital.atlassian.net/browse/CLDC-3171 - need to resolve that conversation.

Reply:
Confirmed at standup that this approach is preferred.